diff --git a/src/CLI/Application.php b/src/CLI/Application.php
index faab87f..fa5c87d 100644
--- a/src/CLI/Application.php
+++ b/src/CLI/Application.php
@@ -109,6 +109,8 @@ private function disableXdebug()
         \ini_set('xdebug.show_exception_trace', 0);
         \ini_set('xdebug.show_error_trace', 0);
 
-        \xdebug_disable();
+        if (\function_exists('xdebug_disable')) {
+            \xdebug_disable();
+        }
     }
 }
